fitbenchmarking/fitbenchmarking
# 882 lm herbert

#### Description of Work

Fixes #882 


#### Testing Instructions

1. Run on a matlab-enabled machine
2. Check that Horace/lm-lsqr works as expected
3. Check that the tests are correct **The regression test needs updating**
4. Check that the docs make sense and are complete.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
